dist/hackerrank.js

```diff
@@ -0,0 +1,33 @@
+class Size {
+    constructor(width, height) {
+        this.width = width;
+        this.height = height;
+    }
+}
+
+class Image {
+    // Add methods here
+    constructor(url, size) {
+        this.url = url;
+        this.width = size.width;
+        this.height = size.height;
+    }
```

Comment:
👉🏼 These classes are not exported and I haven't found any use case for them, It will be nice to remove unnecessary files.

Reply:
Thank you so much @HabibPayenda , I will try my best to apply suggested changes 😄